cfgov/unprocessed/apps/regulations3k/css/reg-wayfinder.less

```diff
@@ -28,35 +30,49 @@
     color: @link-text;
   }
 
-  .content_main,
-  .content_sidebar {
-    padding-top: 15px;
-    padding-bottom: 15px;
+  &_sidebar {
+    display: none;
   }
 
-  .content_main:after {
-    top: 0;
-  }
+  // Tablet and above
+  .respond-to-min(@bp-sm-min, {
+    padding-left: unit(30px / @base-font-size-px, rem);
+    padding-right: unit(30px / @base-font-size-px, rem);
+  });
 
-  .content_sidebar {
-    display: none;
-    .respond-to-min( @bp-med-min, {
+  // Desktop
+  .respond-to-min( @bp-med-min, {
+    &_wrapper {
+      display: grid;
+      grid-template-columns: 1fr 3fr;
+      max-width: 1170px;
```

Comment:
```suggestion
      max-width: 1260px;
```

Reply:
?

Reply:
oh, I hadn't seen your response! This fixes the misalignment, but the other correction is better.